code/game/machinery/alarm.dm

```diff
@@ -56,7 +56,7 @@
 	idle_power_usage = 80
 	active_power_usage = 1000 //For heating/cooling rooms. 1000 joules equates to about 1 degree every 2 seconds for a single tile of air.
 	power_channel = ENVIRON
-	req_access = list(list(access_atmospherics, access_engine_equip))
+	initial_access = list(list(access_atmospherics, access_engine_equip))
```

Comment:
Here's an example where it's a list of lists.

Reply:
Oops. That's a bug. It should only be one list. :) Good catch.